qatrack/faults/signals.py

```
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.core.cache import cache
+from django.db.models.signals import post_delete, post_save
+from django.dispatch import receiver
+
+from qatrack.faults.models import Fault
+
+
+@receiver(post_save, sender=Fault, dispatch_uid="faults_update_faults_cache_post_save")
+@receiver(post_delete, sender=Fault, dispatch_uid="faults_update_faults_cache_post_delete")
+def update_faults_cache(sender, **kwargs):
```

Ruff (0.15.18)

[warning] Unused function argument: `sender`

(ARG001)